src/main/kotlin/com/terraformation/backend/accelerator/db/ModulesImporter.kt

```diff
@@ -51,13 +64,27 @@ class ModulesImporter(
                 .insertInto(MODULES)
                 .set(ID, moduleId)
                 .set(NAME, name)
+                .set(OVERVIEW, overview)
+                .set(PREPARATION_MATERIALS, preparationMaterials)
+                .set(ADDITIONAL_RESOURCES, additionalResources)
+                .set(LIVE_SESSION_DESCRIPTION, liveSessionInfo)
+                .set(ONE_ON_ONE_SESSION_DESCRIPTION, oneOnOneInfo)
+                .set(WORKSHOP_DESCRIPTION, workshopInfo)
+                .set(PHASE_ID, CohortPhase.Phase1FeasibilityStudy)
```

Comment:
Will phase 1 be the only phase with modules? If not, I wonder if we should add a "phase" column to the spreadsheet right now so we don't forget to do it later.

Reply:
I believe there are talks of having Phase 0 modules. 
We should add the column to the spreadsheet but I want to confirm with the products folks first.